### ISSUE TITLE
Style/SpaceAroundOperators doesn't recognize ** power operator

### ISSUE THREAD

**Issue by sparr** [bbatsov/rubocop#4026]
When I do `a+b` the Style/SpaceAroudOperators cop recognizes an offense, but not for `a**b`.

--------

## Expected behavior

`a**b` should trigger the Style/SpaceAroudOperators cop

## Actual behavior

`a**b` is ignored by this cop

## Steps to reproduce the problem

```
echo "puts a+b" > test.rb
echo "puts a**b" >> test.rb
rubocop test.rb
````

## RuboCop version

```
% rubocop -V
0.47.1 (using Parser 2.4.0.0, running on ruby 2.3.0 x86_64-darwin14)
```

**Comment by sparr** [comment]
I later found that SpaceAroundOperators actually insists on ** not having spaces, which seems really weird. This is probably an invalid bug.

**Comment by mikegee** [comment]
Please contribute your thoughts on https://github.com/bbatsov/rubocop/issues/2972 and close this one.

**Comment by sparr** [comment]
Thanks, sorry. Github issue search doesn't think "**" is a legitimate token :(

**Comment by mikegee** [comment]
No worries. I found it by searching "SpaceAroundOperators".
